_data/simple-icons.json

```diff
@@ -1980,6 +1980,11 @@
             "hex": "000000",
             "source": "https://www.farfetch.com/"
         },
+        {
+            "title": "FastAPI",
+            "hex": "009688",
+            "source": "https://github.com/tiangolo/fastapi/blob/6205935323ded4767438ee81623892621b353415/docs/en/docs/img/icon-white-bg.png"
```

Comment:
Why do you link to a raster file?
You even linked to the correct svg files in your PR text... 🤔

Reply:
Hi @runxel, I updated the source :)